package.json

```diff
@@ -32,13 +32,13 @@
     "gulp": "^3.9.0",
     "gulp-eslint": "^0.14.0",
     "gulp-mocha": "^2.1.1",
-    "mocha": "^2.2.5",
-    "node-sass": "^3.3.3",
-    "should": "^6.0.3"
+    "mocha": "^2.3.3",
+    "should": "^7.1.0"
   },
   "dependencies": {
     "lwip": "^0.0.8",
-    "node-sass-utils": "^1.1.1",
+    "node-sass": "^3.4.2",
```

Comment:
I believe it actually has to be a dependency because Layout.js & SpriteMap.js both `require("node-sass")`. More details in [my comment below](https://github.com/sass-eyeglass/eyeglass-spriting/pull/19#issuecomment-167058841).